src/zope/component/tests/test__api.py

```diff
@@ -88,8 +88,7 @@ class IBar(Interface):
             pass
         @implementer(IFoo)
         class Global(object):
-            def __init__(self, context):
-                self.context = context
+            pass
```

Comment:
Maybe have `Global__init__` raise an assertion error instead?  I realize that would require a `pragma:`, but the whole point of the test is that the local registry instantiates `Local` rather than `Global`.